src/components/Learn-Components/Card-Wrapper/Card-Wrapper.component.style.js

```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
```

Comment:
File naming shouldn't be like this, Keep it only `card-wrapper-style.js`. Also, for consistency keep the filenames in a small case in this repo.